Updating branch refs/heads/master
         to eb3e5283fcb07723f07bbd5c9c017e08ad766355 (commit)
       from 042609e9c5a06085eece9803cac6ae96803cfba5 (commit)

commit eb3e5283fcb07723f07bbd5c9c017e08ad766355
Author: Nick Schermer <nick@xfce.org>
Date:   Tue Jun 21 07:36:30 2011 +0200

    Panel: Make preferences dialog work in Gtk+ 2.24.5.
    
    See https://bugzilla.gnome.org/show_bug.cgi?id=576492.

 panel/panel-preferences-dialog.glade |   92 +++++++++++++++++-----------------
 plugins/clock/clock-dialog.glade     |   12 ++--
 2 files changed, 52 insertions(+), 52 deletions(-)

diff --git a/panel/panel-preferences-dialog.glade b/panel/panel-preferences-dialog.glade
index 63f5a2f..8b09f91 100644
--- a/panel/panel-preferences-dialog.glade
+++ b/panel/panel-preferences-dialog.glade
@@ -48,6 +48,52 @@
       </row>
     </data>
   </object>
+    <object class="GtkAdjustment" id="size">
+    <property name="value">16</property>
+    <property name="lower">16</property>
+    <property name="upper">128</property>
+    <property name="step_increment">1</property>
+    <property name="page_increment">10</property>
+  </object>
+  <object class="GtkAdjustment" id="length">
+    <property name="value">25</property>
+    <property name="lower">1</property>
+    <property name="upper">100</property>
+    <property name="step_increment">1</property>
+    <property name="page_increment">10</property>
+  </object>
+  <object class="GtkAdjustment" id="background-alpha">
+    <property name="value">100</property>
+    <property name="upper">100</property>
+    <property name="step_increment">1</property>
+    <property name="page_increment">10</property>
+  </object>
+  <object class="GtkAdjustment" id="enter-opacity">
+    <property name="value">100</property>
+    <property name="upper">100</property>
+    <property name="step_increment">1</property>
+    <property name="page_increment">10</property>
+  </object>
+  <object class="GtkAdjustment" id="leave-opacity">
+    <property name="value">100</property>
+    <property name="upper">100</property>
+    <property name="step_increment">1</property>
+    <property name="page_increment">10</property>
+  </object>
+  <object class="GtkSizeGroup" id="display-sizegroup">
+    <widgets>
+      <widget name="label13"/>
+      <widget name="label1"/>
+    </widgets>
+  </object>
+  <object class="GtkSizeGroup" id="bg-sizegroup">
+    <widgets>
+      <widget name="label7"/>
+      <widget name="label15"/>
+      <widget name="label16"/>
+      <widget name="label19"/>
+    </widgets>
+  </object>
   <object class="XfceTitledDialog" id="dialog">
     <property name="title" translatable="yes">Panel</property>
     <property name="window_position">center-on-parent</property>
@@ -936,50 +982,4 @@
       <action-widget response="1">button-help</action-widget>
     </action-widgets>
   </object>
-  <object class="GtkAdjustment" id="size">
-    <property name="value">16</property>
-    <property name="lower">16</property>
-    <property name="upper">128</property>
-    <property name="step_increment">1</property>
-    <property name="page_increment">10</property>
-  </object>
-  <object class="GtkAdjustment" id="length">
-    <property name="value">25</property>
-    <property name="lower">1</property>
-    <property name="upper">100</property>
-    <property name="step_increment">1</property>
-    <property name="page_increment">10</property>
-  </object>
-  <object class="GtkAdjustment" id="background-alpha">
-    <property name="value">100</property>
-    <property name="upper">100</property>
-    <property name="step_increment">1</property>
-    <property name="page_increment">10</property>
-  </object>
-  <object class="GtkAdjustment" id="enter-opacity">
-    <property name="value">100</property>
-    <property name="upper">100</property>
-    <property name="step_increment">1</property>
-    <property name="page_increment">10</property>
-  </object>
-  <object class="GtkAdjustment" id="leave-opacity">
-    <property name="value">100</property>
-    <property name="upper">100</property>
-    <property name="step_increment">1</property>
-    <property name="page_increment">10</property>
-  </object>
-  <object class="GtkSizeGroup" id="display-sizegroup">
-    <widgets>
-      <widget name="label13"/>
-      <widget name="label1"/>
-    </widgets>
-  </object>
-  <object class="GtkSizeGroup" id="bg-sizegroup">
-    <widgets>
-      <widget name="label7"/>
-      <widget name="label15"/>
-      <widget name="label16"/>
-      <widget name="label19"/>
-    </widgets>
-  </object>
 </interface>
diff --git a/plugins/clock/clock-dialog.glade b/plugins/clock/clock-dialog.glade
index 8b26a17..d746706 100644
--- a/plugins/clock/clock-dialog.glade
+++ b/plugins/clock/clock-dialog.glade
@@ -3,6 +3,12 @@
   <requires lib="gtk+" version="2.14"/>
   <!-- interface-requires libxfce4ui 0.0 -->
   <!-- interface-naming-policy toplevel-contextual -->
+  <object class="GtkAdjustment" id="fuzziness">
+    <property name="upper">3</property>
+    <property name="step_increment">1</property>
+    <property name="page_increment">1</property>
+    <property name="page_size">1</property>
+  </object>
   <object class="XfceTitledDialog" id="dialog">
     <property name="title" translatable="yes">Clock</property>
     <property name="icon_name">gtk-properties</property>
@@ -434,10 +440,4 @@
       </row>
     </data>
   </object>
-  <object class="GtkAdjustment" id="fuzziness">
-    <property name="upper">3</property>
-    <property name="step_increment">1</property>
-    <property name="page_increment">1</property>
-    <property name="page_size">1</property>
-  </object>
 </interface>
_______________________________________________
Xfce4-commits mailing list
Xfce4-commits@xfce.org
http://foo-projects.org/mailman/listinfo/xfce4-commits
